Makefile

```diff
@@ -78,7 +88,7 @@ run: welcome githooks-commit install build
 	@$(NODE) build/bundle-$(CALYPSO_ENV).js
 
 node-version:
-	@NPM_GLOBAL_ROOT=$(shell $(NPM) -g root) $(BIN)/check-node-version
+	@$(BIN)/check-node-version
```

Comment:
it is, though, so far five people I work with ran into semver not actually working at all, and the bin/check-node-version script crashing their attempts to run it, which is why I brought this up.